docs/modules/pyfar.constants.rst

```diff
@@ -5,3 +5,14 @@ pyfar.constants
    :members:
    :undoc-members:
    :show-inheritance:
+
+**Constants**:
+
+.. autodata:: pyfar.constants.standard_atmospheric_pressure
+.. autodata:: pyfar.constants.absolute_zero_celsius
+.. autodata:: pyfar.constants.reference_sound_pressure
+.. autodata:: pyfar.constants.reference_sound_power
+.. autodata:: pyfar.constants.reference_air_temperature_celsius
+.. autodata:: pyfar.constants.reference_speed_of_sound
+.. autodata:: pyfar.constants.standard_air_density
+.. autodata:: pyfar.constants.reference_air_impedance
```

Comment:
In the spirit of the name of the module I would move the constants above the functions in the docs. What do you think?